bucket/gitignore.json

```diff
@@ -7,8 +7,8 @@
     "hash": "733e4e2bece6e84ad1d25589cc6d1f852587f2d4a6ffe74b8268435ae229487a",
     "bin": "gitignore.ps1",
     "checkver": {
-        "url": "https://github.com/lukesampson/psutils",
-        "re": "gitignore\\.ps1[\\S\\s]*?/(?<sha>[0-9a-f]{40})[\\S\\s]*?(\\d+)-(\\d+)-(\\d+)",
+        "url": "https://github.com/lukesampson/psutils/commits/master/gitignore.ps1.atom",
+        "regex": "(?s)(\\d+)-(\\d+)-(\\d+).*?(?<sha>[0-9a-f]{40})",
```

Comment:
Why `\s`?

Reply:
You mean `(?s)`? This trigger make PowerShell search RegEx in single line mode, so `.` matches new line. Ref: https://docs.microsoft.com/en-us/dotnet/standard/base-types/regular-expression-language-quick-reference#regular-expression-options

Reply:
What exactly line you want to match in URL?

Reply:
![ln.ps1.atom](https://user-images.githubusercontent.com/5832170/70037016-cb938b00-15f0-11ea-89bd-5f0542e04e21.png)

Reply:
Then edit it to be more strict. This one could get something you do not want. For example if commit message contain `X-X-X`, then it will be matched instead of date

```suggestion
        "regex": ">(?s)(\\d+)-(\\d+)-(\\d+)T.*?/(?<sha>[0-9a-f]{40})",
```